README.md

```diff
@@ -1,6 +1,8 @@
 # TiDB-Binlog
 
+[![Build Status](https://internal.pingcap.net/idc-jenkins/buildStatus/icon?job=build_tidb_binlog_master)](https://internal.pingcap.net/idc-jenkins/job/build_tidb_binlog_master/)
 [![Coverage Status](https://coveralls.io/repos/github/pingcap/tidb-binlog/badge.svg?branch=HEAD&t=9Zn2om)](https://coveralls.io/github/pingcap/tidb-binlog?branch=HEAD)
```

Comment:
change this `HEAD` to `master` ?

Reply:
I've done that in another branch ... I'll merge it into this one.